string sender
---
string model
uint32 countjoints
string[] inputchannels
string[] outputchannels
